test_doc/template/footer.html

```diff
@@ -0,0 +1,3 @@
+<p style="font-size: 10px; margin-left: 48%">
+  <span class="pageNumber"></span> / <span class="totalPages"></span>
+</p>
```

Comment:
> In order to export my documentation in a specific format expected by my school, I need to include a lot of text and images in the header/footer.

> I could have put everything in the footer/header-template parameter, but it would have been ugly and unmaintainable.

> Actually, I have been asked about the template from file thing before https://github.com/HollowMan6/mdbook-pdf/issues/19#issue-1666170919 I thought Chromium doesn't support rendering complicated header/footer templates. If you are fine with that, can you commit those templates that include text and images directly into this test_doc here (of course remove the sensitive information as well as copyrighted images)?

@skyf0l 

Hmm, can you replace the template with a valid one that can fully demonstrate the feature you are adding here (you can replace the image with any uncopyrighted one)? Which is the one you mentioned that "contains a lot of text and ***images*** in the header/footer, and it can be ugly and unmaintainable if written in one line."

Please don't just copy and paste my example, since it can still be straightforward to maintain in one line, and it contains no ***images***. The reason why I have a strong demand on this is that, as I mentioned earlier https://github.com/HollowMan6/mdbook-pdf/issues/19#issuecomment-1506794601, I thought we don't support rendering complicated header/footer templates (e.g. containing ***images***) The feature you are adding here will be meaningless if the template can still be easily maintained in one line. Thanks!